InventoryComponents/InventoryHUD/RelicDisplay.gd

```diff
@@ -0,0 +1,7 @@
+extends Control
+class_name RelicDisplay
```

Comment:
what's the difference between `RelicDisplay` and `RelicComponent` ? one is the UI and the other the data ?